common/test/acceptance/pages/lms/video/video.py

```diff
@@ -222,11 +223,26 @@ def _is_element_present():
 
             """
             is_present = self.q(css=selector).present
+            # TODO! Better hls source url detection. Currently this is workaround
```

Comment:
Are we going to do this in this PR otherwise I think we should add the JIRA refence so that when doing this it can be referenced asily.